tests/kafkatest/services/console_consumer.py

```diff
@@ -151,7 +151,10 @@ def prop_file(self, node):
     def start_cmd(self, node):
         """Return the start command appropriate for the given node."""
         args = self.args.copy()
-        args['zk_connect'] = self.kafka.zk_connect_setting()
+        if self.new_consumer:
+            args['broker_list'] = self.kafka.bootstrap_servers(self.security_config.security_protocol)
+        else:
+            args['zk_connect'] = self.kafka.zk_connect_setting()
```

Comment:
This is a slight behavior change for the old consumer, right?  Previously we set both `broker_list` and `zk_connect`, but now we set only `zk_connect`. Can we keep setting both for now? (I agree we should skip setting zk_connect for the new consumer)